### DISCUSSION TITLE
Help with cdock on Monterey (Intel)

### DISCUSSION THREAD

**Discussion by ffabbroc** [w0lfschild/cDock2#323] [Q&A]
Can anyone help with getting cdock to work on Monterey (Intel)?

I see some discord server invites to get a working version but they are all expired.

Thanks,
Francesco

**Comment by rklbln** [comment]
I bought the program. nothing to work. Monterey needs to install an help program with user credential, but after input the password no0thing is working. Maybe special rights needs. SIP is disabled.
